composer.json

```diff
@@ -230,7 +230,7 @@
         "symfony/runtime": "^5.4.21 || ^6.0",
         "symfony/web-profiler-bundle": "^5.4.21 || ^6.0",
         "symplify/monorepo-builder": "^11.0",
-        "vimeo/psalm": "^4.19"
```

Comment:
There are problems with `vimeo/psalm` `5.10+`. It throws errors, and we're unable to suppress them using the configuration. There's a weird bug, we should report it as a next step.